tools/build_pip_package.py

```diff
@@ -0,0 +1,51 @@
+#!/bin/python3
+# Copyright 2023 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from datetime import datetime, timezone
+import os
+import pkg_resources
+
+from setuptools import setup
+
+VERSION = '0.1.0.dev' + datetime.now(tz=timezone.utc).strftime('%Y%m%d%H%M')
+
+SHARED_OBJECT_MANIFEST_PATH = os.path.join(
+    os.path.dirname(__file__), 'shared_object_list.txt'
+)
```

Comment:
We should be able to use https://github.com/bazelbuild/bazel/blob/master/tools/python/runfiles/runfiles.py to go from a repository-relative path (`gematria/tools/shared_object_list.txt`) to an actual file in runfiles.